src/Adapter/MSTest.TestAdapter/Constants.cs

```diff
@@ -10,6 +10,36 @@ namespace Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter;
 /// </summary>
 internal static class Constants
 {
+    /// <summary>
+    /// Assembly.
+    /// </summary>
+    internal const string AssemblyFixtures = "Assembly";
```

Comment:
For all these new entries, please use a naming that conveys the meaning. Is it a prefix? suffix? Category name? Entry display name?

Reply:
Updated,